torch/distributed/_tensor/ops/tensor_ops.py

```diff
@@ -359,6 +360,28 @@ def replica_only_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType
     return OpStrategy([PlacementStrategy(replicate_spec)])
 
 
+@register_op_strategy(
+    [aten.scatter_.value, aten.scatter.value, aten.scatter_.src, aten.scatter.src],
+    schema_info=RuntimeSchemaInfo(1),
+)
+def scatter_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType:
+    input_strategy = cast(OpStrategy, op_schema.args_schema[0])
+    single_mesh_dim_strategies = []
+
+    # placement list stores placements of [output, input, index, src]
+    # first we always have replicate all for inputs and output
+    all_replicate: List[Placement] = [Replicate()] * 4
```

Comment:
For `aten.scatter_.value` and `aten.scatter.value`, `src` is a value rather than a tensor, although I think the code here should still work in those cases. Can we comment here and add a test case where `src` is a float number?